dev/TeachingTip/TeachingTip.cpp

```diff
@@ -178,10 +183,10 @@ void TeachingTip::CreateLightDismissIndicatorPopup()
     }
 }
 
-void TeachingTip::UpdateTail()
+bool TeachingTip::UpdateTail()
 {
```

Comment:
this function seems like it does much more than "UpdateTail", is there a better name for it?

Reply:
I don't think this does much more than update the tail. I does not bubble up the info about if the tip fits from the call to determineEffectivePlacement

---
In reply to: [269289458](https://github.com/Microsoft/microsoft-ui-xaml/pull/480#discussion_r269289458) [](ancestors = 269289458)